src/sagemaker/jumpstart/factory/model.py

```diff
@@ -160,6 +165,16 @@ def _add_model_version_to_kwargs(
     return kwargs
 
 
+def _log_model_type(kwargs: JumpStartModelInitKwargs) -> None:
```

Comment:
confusing function name/docstring, you're only loading if the model type is "proprietary"